extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegeObject.scala

```diff
@@ -66,14 +66,18 @@ object PrivilegeObject {
       table: Table,
       columns: Seq[String] = Nil,
       actionType: PrivilegeObjectActionType = PrivilegeObjectActionType.OTHER): PrivilegeObject = {
-    new PrivilegeObject(
-      TABLE_OR_VIEW,
-      actionType,
-      table.database.orNull,
-      table.table,
-      columns,
-      table.owner,
-      table.catalog)
+    if (table.table.contains("/")) {
```

Comment:
Let's differentiate tables from URLs in command desc in a consistent way. which means we shall add a new UriDesc to the command

BTW, '/' is an invalid character for table names mostly, but there are exceptions, i.e. escaping by backticks

Reply:
To determine whether it is a path-based table, I can refer to the implementation of delta lake. 
But add a new UriDesc to the command should do it when needed, e.g. OPTIMIZE 'path', otherwise the match will be repeated. What do you think？

Reply:
Repetition is acceptable in this case. Let's keep it as simple as possible for the caller side

Reply:
Okay, I will make changes later